### ISSUE TITLE
generate apk `react-native-voice:verifyReleaseResources` error

### ISSUE THREAD

**Issue by MiguelGrability** [react-native-community/react-native-voice#177]
Hi, guys

I have been trying to generate an APK with RN 0.59.5 and I couldn't, I got this error

`Task :react-native-voice:verifyReleaseResources FAILED`

```
* What went wrong:
Execution failed for task ':react-native-voice:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
  error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
```

I have been working in this project with RN version 0.56, I have upgraded to 0.59 and I am getting the above error

**Comment by MiguelGrability** [comment]
I've been working around, the problem is that android is too outdated, I had to upgrade graddle in my node_modules to be able to generate a signed apk.

I hope that this issue be fixed soon.

**Comment by lfoliveir4** [comment]
> I've been working around, the problem is that android is too outdated, I had to upgrade graddle in my node_modules to be able to generate a signed apk.
> 
> I hope that this issue be fixed soon.

Up, I have the same problem.
What was your solution? Which version did you put your gradle modules in?
Was the app inconsistent?

**Comment by MiguelGrability** [comment]
> > I've been working around, the problem is that android is too outdated, I had to upgrade graddle in my node_modules to be able to generate a signed apk.
> > I hope that this issue be fixed soon.
> 
> Up, I have the same problem.
> What was your solution? Which version did you put your gradle modules in?
> Was the app inconsistent?

I have opened my `node_modules/react-native-voice/android` folder on AndroidStudio then upgrade gradle like my project

something like this

```
apply plugin: 'com.android.library'

android {
    compileSdkVersion 28

    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 28
        versionCode 1
        versionName "1.0"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

buildscript {
    repositories {
        jcenter()
        google()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.3.2'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        jcenter()
        google()
    }
}

dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
    // testCompile 'junit:junit:4.12'
   implementation 'com.android.support:appcompat-v7:28.0.0'
   implementation 'com.facebook.react:react-native:+'
}
```

replace in gradle.properties this
`distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip`

after that I was able to sync `react-native-voice` android project and I could build my release

**Comment by lfoliveir4** [comment]
> > > I've been working around, the problem is that android is too outdated, I had to upgrade graddle in my node_modules to be able to generate a signed apk.
> > > I hope that this issue be fixed soon.
> > 
> > 
> > Up, I have the same problem.
> > What was your solution? Which version did you put your gradle modules in?
> > Was the app inconsistent?
> 
> I have opened my `node_modules/react-native-voice/android` folder on AndroidStudio then upgrade gradle like my project
> 
> something like this
> 
> ```
> apply plugin: 'com.android.library'
> 
> android {
>     compileSdkVersion 28
> 
>     defaultConfig {
>         minSdkVersion 16
>         targetSdkVersion 28
>         versionCode 1
>         versionName "1.0"
>     }
>     buildTypes {
>         release {
>             minifyEnabled false
>             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
>         }
>     }
> }
> 
> buildscript {
>     repositories {
>         jcenter()
>         google()
>     }
>     dependencies {
>         classpath 'com.android.tools.build:gradle:3.3.2'
> 
>         // NOTE: Do not place your application dependencies here; they belong
>         // in the individual module build.gradle files
>     }
> }
> 
> allprojects {
>     repositories {
>         jcenter()
>         google()
>     }
> }
> 
> dependencies {
>    implementation fileTree(dir: 'libs', include: ['*.jar'])
>     // testCompile 'junit:junit:4.12'
>    implementation 'com.android.support:appcompat-v7:28.0.0'
>    implementation 'com.facebook.react:react-native:+'
> }
> ```
> 
> replace in gradle.properties this
> `distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip`
> 
> after that I was able to sync `react-native-voice` android project and I could build my release

![Captura de Tela 2019-05-06 às 16 42 10](https://user-images.githubusercontent.com/32423942/57250495-ee8b9b80-701d-11e9-8a02-09b9a2f37b6a.png)
this?

**Comment by MiguelGrability** [comment]
I'm sorry @lfoliveir4 I mean in `gradle-wrapper.properties`

```
#Mon Dec 28 10:00:20 PST 2015
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```

**Comment by lfoliveir4** [comment]
> I'm sorry @lfoliveir4 I mean in `gradle-wrapper.properties`
> 
> ```
> #Mon Dec 28 10:00:20 PST 2015
> distributionBase=GRADLE_USER_HOME
> distributionPath=wrapper/dists
> zipStoreBase=GRADLE_USER_HOME
> zipStorePath=wrapper/dists
> distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
> ```

Thank you very much!!!
this method worked for me !!!
Let's wait for correction !!!

**Comment by matmunn** [comment]
This should be fixed by #180. Just mentioning it here so people can see where it's at. Thanks @lfoliveir4

**Comment by Desintegrator** [comment]
consider reopening this issue
package still has old android sdk
